redux-kotlin/src/commonMain/kotlin/org/reduxkotlin/CreateStore.kt

```diff
@@ -246,3 +246,26 @@ public fun <State> createStore(
         override val replaceReducer = ::replaceReducer
     }
 }
+
+/**
+ * Creates a [TypedStore]. For further details see the matching [createStore].
+ */
+public inline fun <State, reified Action : Any> createTypedStore(
+    crossinline reducer: TypedReducer<State, Action>,
+    preloadedState: State,
+    noinline enhancer: StoreEnhancer<State>? = null
+): TypedStore<State, Action> {
+    val store = createStore(
+        reducer = typedReducer(reducer),
+        preloadedState,
+        enhancer,
+    )
+    return object : TypedStore<State, Action> {
+        override val getState: GetState<State> = store.getState
+        override var dispatch: TypedDispatcher<Action> = store.dispatch
+        override val subscribe: (StoreSubscriber) -> StoreSubscription = store.subscribe
+        override val replaceReducer: (TypedReducer<State, Action>) -> Unit = {
+            store.replaceReducer(typedReducer(it))
+        }
+    }
```

Comment:
Can this use `asTyped()`?
 = createStore(
        reducer = typedReducer(reducer),
        preloadedState,
        enhancer,
    ).asTyped()

Reply:
Good spot!